pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,10 @@ type Mux interface {
 type APIGroupVersion struct {
 	Storage map[string]rest.Storage
 
+	// Root is the APIPrefix under which this is being served.  It can also be the APIPrefix/APIGroup that is being served
```

Comment:
@smarterclayton, yes, I understand. What I tried to say was the fields in APIGroupVersion are populated to many places so it would be difficult to rebase #14156 on this change. I agree we should separate group and version where we can.